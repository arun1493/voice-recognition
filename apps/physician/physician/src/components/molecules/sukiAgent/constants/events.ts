import { ClientReceiveEventTypes } from "@suki-web/suki-business";

import {
  handleCompositionCreationInNotepad,
  handleCreateCompositionAcknowledgement,
  handleIntent,
  handleResourceExhaustedError,
  handleTurnOffAgent,
  handleTurnOnAgent
} from "../eventHandlers";

const AGENT_TURN_ON_EVENT = {
  type: ClientReceiveEventTypes.TURN_ON_AGENT,
  handler: handleTurnOnAgent
};

const AGENT_TURN_OFF_EVENT = {
  type: ClientReceiveEventTypes.TURN_OFF_AGENT,
  handler: handleTurnOffAgent
};

const INTENT_EVENT = {
  type: ClientReceiveEventTypes.INTENT,
  handler: handleIntent
};

const COMPOSITION_CREATED_IN_NOTEPAD_EVENT = {
  type: ClientReceiveEventTypes.COMPOSITION_CREATED_IN_NOTEPAD,
  handler: handleCompositionCreationInNotepad
};

const CREATE_COMPOSITION_ACKNOWLEDGEMENT_EVENT = {
  type: ClientReceiveEventTypes.CREATE_COMPOSITION_ACKNOWLEDGEMENT,
  handler: handleCreateCompositionAcknowledgement
};

const RESOURCE_EXHAUSTED_ERROR_EVENT = {
  type: ClientReceiveEventTypes.RESOURCE_EXHAUSTED_ERROR,
  handler: handleResourceExhaustedError
};

const EVENTS = [
  AGENT_TURN_OFF_EVENT,
  AGENT_TURN_ON_EVENT,
  COMPOSITION_CREATED_IN_NOTEPAD_EVENT,
  CREATE_COMPOSITION_ACKNOWLEDGEMENT_EVENT,
  INTENT_EVENT,
  RESOURCE_EXHAUSTED_ERROR_EVENT
];

export { EVENTS };
