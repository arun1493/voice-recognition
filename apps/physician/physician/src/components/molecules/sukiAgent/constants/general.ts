import { ScreenSize } from "@suki-web/suki-ui";
import { Paths } from "@suki-web/suki-business";

const HORIZONTAL_OFFSET: Record<ScreenSize, number> = {
  xxs: 0,
  xs: 0,
  sm: -164,
  md: -184,
  lg: -224,
  xl: -260
};

const HIDDEN_AGENT_ROUTES: string[] = [
  Paths.Authentication,
  Paths.BusinessAssociateAgreement,
  Paths.ForgotPassword,
  Paths.Login,
  Paths.Password,
  Paths.Payments,
  Paths.PaymentSuccess,
  Paths.Register,
  Paths.ScheduleImport,
  Paths.TechnologyServicesAgreement,
  Paths.UnSupportedDeviceOrBrowser,
  Paths.XLiteSuccess,
  Paths.WindowsUpdate
];

const DISABLED_AGENT_ROUTES: string[] = [Paths.Settings, Paths.Scripts, Paths.NoteTypes];

export { HIDDEN_AGENT_ROUTES, HORIZONTAL_OFFSET, DISABLED_AGENT_ROUTES };
