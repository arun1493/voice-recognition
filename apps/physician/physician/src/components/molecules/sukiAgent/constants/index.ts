export * from "./sounds";
export * from "./events";
export * from "./general";
