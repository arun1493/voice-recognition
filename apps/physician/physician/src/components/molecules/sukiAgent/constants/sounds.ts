enum AgentSounds {
  END_OF_UTTERANCE,
  ERROR,
  EXIT_DICTATION
}

const AgentSoundPaths = {
  [AgentSounds.END_OF_UTTERANCE]: "../../../../assets/sounds/end_of_utterance.mp3",
  [AgentSounds.ERROR]: "../../../../assets/sounds/error.mp3",
  [AgentSounds.EXIT_DICTATION]: "../../../../assets/sounds/exit_dictation.mp3"
};

export { AgentSounds, AgentSoundPaths };
