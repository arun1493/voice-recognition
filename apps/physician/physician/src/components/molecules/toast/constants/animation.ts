const TOAST_TRANSITION_DURATION = 300;

export { TOAST_TRANSITION_DURATION };
