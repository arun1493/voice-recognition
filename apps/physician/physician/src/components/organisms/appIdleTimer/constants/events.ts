enum TimeOutEvents {
  STAY_SIGNED_IN = "STAY_SIGNED_IN"
}

export { TimeOutEvents };
