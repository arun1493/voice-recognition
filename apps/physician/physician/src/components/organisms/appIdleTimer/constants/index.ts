export * from "./events";
export * from "./timeOuts";
