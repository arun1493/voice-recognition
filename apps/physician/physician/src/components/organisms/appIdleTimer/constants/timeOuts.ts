const INACTIVITY_TIMEOUT = 1000 * 60 * 59.5; // 59.5 minutes
const INACTIVITY_PROMPT_TIMEOUT = 1000 * 60 * 0.5; // 0.5 minutes

export { INACTIVITY_PROMPT_TIMEOUT, INACTIVITY_TIMEOUT };
