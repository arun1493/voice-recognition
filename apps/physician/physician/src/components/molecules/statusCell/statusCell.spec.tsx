import { NoteStatus } from "@suki-web/suki-ui";
import "@testing-library/jest-dom/extend-expect";
import { cleanup, RenderResult } from "@testing-library/react";

import { PAGES } from "../../../constants";
import { StatusCell, StatusCellProps } from ".";
import { componentSetupWithStore } from "../../../hocs";
import { shouldMatchSnapshot } from "../../../helpers/snapshot.spec";

const props: StatusCellProps = {
  status: NoteStatus.USER_IS_DONE
};

const setup = componentSetupWithStore(() => <StatusCell {...props} />);

describe("Patient Status", () => {
  let component: RenderResult;

  beforeEach(() => {
    component = setup();
  });

  afterEach(() => {
    cleanup();
  });

  it("should render successfully", () => {
    const { baseElement } = component;
    expect(baseElement).toBeTruthy();
  });

  it("should render correct status", () => {
    const { baseElement } = component;
    const svg = baseElement.getElementsByTagName("svg");
    expect(svg).toHaveLength(1);
  });

  it("should match snapshot - selectedClose icon", () => {
    const props: StatusCellProps = {
      hasDeleteButton: true,
      onDelete: jest.fn
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - listArrow icon", () => {
    const props: StatusCellProps = {
      type: PAGES.SETTINGS
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - check icon", () => {
    const props: StatusCellProps = {
      type: PAGES.UNFINISHED_NOTES,
      status: NoteStatus.USER_IS_DONE
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - dot icon", () => {
    const props: StatusCellProps = {
      type: PAGES.UNFINISHED_NOTES,
      status: NoteStatus.SUBMITTED_TO_EMR
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - listUnfinishedNote icon", () => {
    const props: StatusCellProps = {
      type: PAGES.UNFINISHED_NOTES,
      status: NoteStatus.INCOMPLETE
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - listSignNote icon", () => {
    const props: StatusCellProps = {
      type: PAGES.UNFINISHED_NOTES,
      status: NoteStatus.NEED_DOCTOR_SIGN_OFF
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - clock icon", () => {
    const props: StatusCellProps = {
      status: NoteStatus.USER_IS_DONE
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - exclamation icon", () => {
    const props: StatusCellProps = {
      status: NoteStatus.INCOMPLETE
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - check icon", () => {
    const props: StatusCellProps = {
      status: NoteStatus.SUBMITTED_TO_EMR
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - chart icon", () => {
    const props: StatusCellProps = {
      isChart: true
    };
    shouldMatchSnapshot(StatusCell, props);
  });

  it("should match snapshot - empty cell", () => {
    shouldMatchSnapshot(StatusCell);
  });
});
