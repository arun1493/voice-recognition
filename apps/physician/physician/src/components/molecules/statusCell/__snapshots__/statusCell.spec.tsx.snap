// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Patient Status should match snapshot - chart icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    fill="none"
    height={32}
    viewBox="0 0 32 34"
    width={32}
  >
    <path
      d="M21.0392 3.36612C21.5274 2.87796 22.3188 2.87796 22.807 3.36612C23.2951 3.85427 23.2951 4.64573 22.807 5.13388L14.6443 13.2966C14.1562 13.7847 13.3647 13.7847 12.8765 13.2966L9.36612 9.78614C8.87796 9.29799 8.87796 8.50653 9.36612 8.01837C9.85427 7.53022 10.6457 7.53022 11.1339 8.01837L13.7604 10.6449L21.0392 3.36612Z"
      fill="#2e5f62"
    />
    <path
      d="M4.27539 23.5059C3.86523 23.5059 3.49805 23.5801 3.17383 23.7285C2.85352 23.877 2.58203 24.0918 2.35938 24.373C2.13672 24.6504 1.9668 24.9883 1.84961 25.3867C1.73242 25.7812 1.67383 26.2266 1.67383 26.7227C1.67383 27.3867 1.76758 27.959 1.95508 28.4395C2.14258 28.916 2.42773 29.2832 2.81055 29.541C3.19336 29.7949 3.67773 29.9219 4.26367 29.9219C4.62695 29.9219 4.98047 29.8867 5.32422 29.8164C5.66797 29.7461 6.02344 29.6504 6.39062 29.5293V30.7246C6.04297 30.8613 5.69141 30.9609 5.33594 31.0234C4.98047 31.0859 4.57031 31.1172 4.10547 31.1172C3.23047 31.1172 2.50391 30.9355 1.92578 30.5723C1.35156 30.209 0.921875 29.6973 0.636719 29.0371C0.355469 28.377 0.214844 27.6035 0.214844 26.7168C0.214844 26.0684 0.304688 25.4746 0.484375 24.9355C0.664062 24.3965 0.925781 23.9316 1.26953 23.541C1.61328 23.1465 2.03711 22.8438 2.54102 22.6328C3.04883 22.418 3.62891 22.3105 4.28125 22.3105C4.71094 22.3105 5.13281 22.3594 5.54688 22.457C5.96484 22.5508 6.35156 22.6855 6.70703 22.8613L6.21484 24.0215C5.91797 23.8809 5.60742 23.7598 5.2832 23.6582C4.95898 23.5566 4.62305 23.5059 4.27539 23.5059ZM9.4668 21.8828V24.1738C9.4668 24.4121 9.45898 24.6445 9.44336 24.8711C9.43164 25.0977 9.41797 25.2734 9.40234 25.3984H9.47852C9.61523 25.1719 9.7832 24.9863 9.98242 24.8418C10.1816 24.6934 10.4043 24.582 10.6504 24.5078C10.9004 24.4336 11.166 24.3965 11.4473 24.3965C11.9434 24.3965 12.3652 24.4805 12.7129 24.6484C13.0605 24.8125 13.3262 25.0684 13.5098 25.416C13.6973 25.7637 13.791 26.2148 13.791 26.7695V31H12.4141V27.0273C12.4141 26.5234 12.3105 26.1465 12.1035 25.8965C11.8965 25.6426 11.5762 25.5156 11.1426 25.5156C10.7246 25.5156 10.3926 25.6035 10.1465 25.7793C9.9043 25.9512 9.73047 26.207 9.625 26.5469C9.51953 26.8828 9.4668 27.293 9.4668 27.7773V31H8.08984V21.8828H9.4668ZM18.2617 24.3906C19.082 24.3906 19.7012 24.5723 20.1191 24.9355C20.541 25.2988 20.752 25.8652 20.752 26.6348V31H19.7734L19.5098 30.0801H19.4629C19.2793 30.3145 19.0898 30.5078 18.8945 30.6602C18.6992 30.8125 18.4727 30.9258 18.2148 31C17.9609 31.0781 17.6504 31.1172 17.2832 31.1172C16.8965 31.1172 16.5508 31.0469 16.2461 30.9062C15.9414 30.7617 15.7012 30.543 15.5254 30.25C15.3496 29.957 15.2617 29.5859 15.2617 29.1367C15.2617 28.4688 15.5098 27.9668 16.0059 27.6309C16.5059 27.2949 17.2598 27.1094 18.2676 27.0742L19.3926 27.0332V26.6934C19.3926 26.2441 19.2871 25.9238 19.0762 25.7324C18.8691 25.541 18.5762 25.4453 18.1973 25.4453C17.873 25.4453 17.5586 25.4922 17.2539 25.5859C16.9492 25.6797 16.6523 25.7949 16.3633 25.9316L15.918 24.959C16.2344 24.791 16.5938 24.6543 16.9961 24.5488C17.4023 24.4434 17.8242 24.3906 18.2617 24.3906ZM19.3867 27.9004L18.5488 27.9297C17.8613 27.9531 17.3789 28.0703 17.1016 28.2812C16.8242 28.4922 16.6855 28.7812 16.6855 29.1484C16.6855 29.4688 16.7812 29.7031 16.9727 29.8516C17.1641 29.9961 17.416 30.0684 17.7285 30.0684C18.2051 30.0684 18.5996 29.9336 18.9121 29.6641C19.2285 29.3906 19.3867 28.9902 19.3867 28.4629V27.9004ZM25.9668 24.3906C26.0762 24.3906 26.1953 24.3965 26.3242 24.4082C26.4531 24.4199 26.5645 24.4355 26.6582 24.4551L26.5293 25.7441C26.4473 25.7207 26.3457 25.7031 26.2246 25.6914C26.1074 25.6797 26.002 25.6738 25.9082 25.6738C25.6621 25.6738 25.4277 25.7148 25.2051 25.7969C24.9824 25.875 24.7852 25.9961 24.6133 26.1602C24.4414 26.3203 24.3066 26.5215 24.209 26.7637C24.1113 27.0059 24.0625 27.2871 24.0625 27.6074V31H22.6797V24.5137H23.7578L23.9453 25.6562H24.0098C24.1387 25.4258 24.2988 25.2148 24.4902 25.0234C24.6816 24.832 24.9004 24.6797 25.1465 24.5664C25.3965 24.4492 25.6699 24.3906 25.9668 24.3906ZM30.2734 30.0039C30.4531 30.0039 30.6309 29.9883 30.8066 29.957C30.9824 29.9219 31.1426 29.8809 31.2871 29.834V30.877C31.1348 30.9434 30.9375 31 30.6953 31.0469C30.4531 31.0938 30.2012 31.1172 29.9395 31.1172C29.5723 31.1172 29.2422 31.0566 28.9492 30.9355C28.6562 30.8105 28.4238 30.5977 28.252 30.2969C28.0801 29.9961 27.9941 29.5801 27.9941 29.0488V25.5625H27.1094V24.9473L28.0586 24.4609L28.5098 23.0723H29.377V24.5137H31.2344V25.5625H29.377V29.0312C29.377 29.3594 29.459 29.6035 29.623 29.7637C29.7871 29.9238 30.0039 30.0039 30.2734 30.0039Z"
      fill="#2e5f62"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - check icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    fill="none"
    height={18}
    viewBox="0 0 15 11"
    width={18}
  >
    <path
      d="M12.0392 0.366117C12.5274 -0.122038 13.3188 -0.122039 13.807 0.366116C14.2951 0.854272 14.2951 1.64573 13.807 2.13388L5.64431 10.2966C5.15615 10.7847 4.36469 10.7847 3.87654 10.2966L0.366116 6.78614C-0.122039 6.29799 -0.122038 5.50653 0.366116 5.01837C0.854272 4.53022 1.64573 4.53022 2.13388 5.01837L4.76043 7.6449L12.0392 0.366117Z"
      fill="#2e5f62"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - check icon 2`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    fill="none"
    height={24}
    viewBox="0 0 15 11"
    width={24}
  >
    <path
      d="M12.0392 0.366117C12.5274 -0.122038 13.3188 -0.122039 13.807 0.366116C14.2951 0.854272 14.2951 1.64573 13.807 2.13388L5.64431 10.2966C5.15615 10.7847 4.36469 10.7847 3.87654 10.2966L0.366116 6.78614C-0.122039 6.29799 -0.122038 5.50653 0.366116 5.01837C0.854272 4.53022 1.64573 4.53022 2.13388 5.01837L4.76043 7.6449L12.0392 0.366117Z"
      fill="#2e5f62"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - clock icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    fill="none"
    height={24}
    viewBox="0 0 21 23"
    width={24}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M18.3851 3.58965L17.6851 3.08845C17.5735 3.00861 17.4605 2.93077 17.3459 2.85497C17.198 2.7571 17.0476 2.66265 16.8948 2.57172C15.2172 1.57323 13.2516 0.998535 11.1468 0.998535C4.99061 0.998535 0 5.9234 0 11.9985C0 18.0737 4.99061 22.9985 11.1468 22.9985C15.2066 22.9985 18.7605 20.8574 20.7108 17.6523C20.8979 17.3447 20.8746 17.0405 20.7071 16.7726C20.5276 16.4856 20.1741 16.2422 19.7163 16.2012C19.4754 16.1796 19.16 16.3183 18.9476 16.6634C17.35 19.2601 14.4546 20.9985 11.1468 20.9985C6.10992 20.9985 2.0267 16.9691 2.0267 11.9985C2.0267 7.02797 6.10992 2.99854 11.1468 2.99854C12.4543 2.99854 13.7 3.27065 14.8269 3.76157C15.1934 3.92123 15.5471 4.10393 15.8862 4.30779L16.9868 4.96955L18.3851 3.58965Z"
      fill="#999999"
    />
    <path
      d="M9.26316 7.9459C9.26316 7.30642 9.78157 6.78801 10.4211 6.78801C11.0605 6.78801 11.5789 7.30642 11.5789 7.9459V13.7354C11.5789 14.3749 11.0605 14.8933 10.4211 14.8933C9.78157 14.8933 9.26316 14.3749 9.26316 13.7354V7.9459Z"
      fill="#999999"
    />
    <path
      d="M13.8947 12.5775C14.5342 12.5775 15.0526 13.0959 15.0526 13.7354C15.0526 14.3749 14.5342 14.8933 13.8947 14.8933H10.4211C9.78157 14.8933 9.26316 14.3749 9.26316 13.7354C9.26316 13.0959 9.78157 12.5775 10.4211 12.5775H13.8947Z"
      fill="#999999"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - dot icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    data-cy="status-cell"
    height={18}
    viewBox="0 0 24 24"
    width={18}
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="12.5"
      cy="12.5"
      fill="#ffa015"
      fillRule="evenodd"
      opacity="1"
      r="5.5"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - dot icon 2`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    data-cy="status-cell"
    height={18}
    viewBox="0 0 24 24"
    width={18}
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="12.5"
      cy="12.5"
      fill="#ffa015"
      fillRule="evenodd"
      opacity="1"
      r="5.5"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - empty cell 1`] = `
<div
  className="statusCellstyled__StyledEmptyCell-sc-3olh8y-0 gmDhJw"
  data-cy="status-cell"
/>
`;

exports[`Patient Status should match snapshot - emr icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    fill="none"
    height={32}
    viewBox="0 0 32 35"
    width={32}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M21.0392 4.36465C21.5274 3.8765 22.3188 3.8765 22.807 4.36465C23.2951 4.85281 23.2951 5.64426 22.807 6.13242L14.6443 14.2951C14.1562 14.7832 13.3647 14.7832 12.8765 14.2951L9.36612 10.7847C8.87796 10.2965 8.87796 9.50506 9.36612 9.01691C9.85427 8.52875 10.6457 8.52875 11.1339 9.01691L13.7604 11.6434L21.0392 4.36465Z"
      fill="#2E5F62"
    />
    <path
      d="M8.97656 31.9985H4.125V23.4321H8.97656V24.6157H5.53125V26.9419H8.75977V28.1196H5.53125V30.8091H8.97656V31.9985ZM14.5254 31.9985L12.0586 24.8442H12.0059C12.0176 25.0044 12.0312 25.2231 12.0469 25.5005C12.0625 25.7778 12.0762 26.0747 12.0879 26.3911C12.0996 26.7075 12.1055 27.0103 12.1055 27.2993V31.9985H10.8281V23.4321H12.8027L15.1758 30.27H15.2109L17.666 23.4321H19.6348V31.9985H18.293V27.229C18.293 26.9673 18.2969 26.6841 18.3047 26.3794C18.3164 26.0747 18.3281 25.7856 18.3398 25.5122C18.3555 25.2388 18.3672 25.02 18.375 24.856H18.3281L15.7676 31.9985H14.5254ZM24.2988 23.4321C25.0371 23.4321 25.6465 23.522 26.127 23.7017C26.6113 23.8813 26.9707 24.1548 27.2051 24.522C27.4434 24.8892 27.5625 25.356 27.5625 25.9224C27.5625 26.3442 27.4844 26.7036 27.3281 27.0005C27.1719 27.2974 26.9668 27.5435 26.7129 27.7388C26.459 27.9341 26.1875 28.0903 25.8984 28.2075L28.3301 31.9985H26.7422L24.6738 28.5649H23.2969V31.9985H21.8906V23.4321H24.2988ZM24.2051 24.604H23.2969V27.4048H24.2695C24.9219 27.4048 25.3945 27.2837 25.6875 27.0415C25.9844 26.7993 26.1328 26.4419 26.1328 25.9692C26.1328 25.4731 25.9746 25.1216 25.6582 24.9146C25.3457 24.7075 24.8613 24.604 24.2051 24.604Z"
      fill="#2E5F62"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - exclamation icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    data-cy="status-cell"
    fill="none"
    height={24}
    viewBox="0 0 24 25"
    width={24}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clipRule="evenodd"
      d="M11.5278 3.99854C10.6891 3.99854 10.0134 4.68614 10.028 5.52467L10.1848 14.5247C10.1991 15.3428 10.8664 15.9985 11.6846 15.9985H12.3208C13.1392 15.9985 13.8065 15.3426 13.8206 14.5243L13.9752 5.5243C13.9896 4.68591 13.3139 3.99854 12.4754 3.99854H11.5278ZM11.1172 17.9985C10.501 17.9985 10.0014 18.5358 10.0014 19.1985V20.7985C10.0014 21.4613 10.501 21.9985 11.1172 21.9985H12.8856C13.5018 21.9985 14.0014 21.4613 14.0014 20.7985V19.1985C14.0014 18.5358 13.5018 17.9985 12.8856 17.9985H11.1172Z"
      fill="#ffa015"
      fillRule="evenodd"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - listArrow icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    data-cy="status-cell"
    height={16}
    viewBox="0 0 9 16"
    width={16}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M2.26479 1.17727C1.86176 0.774242 1.20831 0.774242 0.80528 1.17727C0.402247 1.58031 0.402248 2.23375 0.80528 2.63678L6.1685 8L0.80528 13.3632C0.402247 13.7663 0.402247 14.4197 0.80528 14.8227C1.20831 15.2258 1.86176 15.2258 2.26479 14.8227L8.35776 8.72975C8.76079 8.32672 8.76079 7.67328 8.35776 7.27025L2.26479 1.17727Z"
      fill="#999999"
      fillRule="evenodd"
      opacity="1"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - listSignNote icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    data-cy="status-cell"
    height={24}
    viewBox="0 0 24 24"
    width={24}
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      fill="none"
      fillRule="evenodd"
    >
      <path
        d="M8.112 20.303h11.1"
        stroke="#2e5f62"
        strokeLinecap="round"
        strokeWidth="1.572"
      />
      <path
        d="M8.017 17.21c-.387.394-.799.754-1.236 1.08-.547.408-1.275.892-2.184 1.45l-.573-.48c.356-.961.707-1.76 1.053-2.4.346-.638.71-1.219 1.09-1.742l4.224-5.034 2.135 1.791-4.224 5.034zm2.321-2.737 4.949-5.897 1.972-2.35m-8.61 5.918 6.52-7.672M11.753 8.77l1.924 1.615"
        stroke="#2e5f62"
        strokeWidth="1.572"
      />
      <path
        d="m15.067 4.593.568-.677c.423-.504 2.51 1.247 2.087 1.751-.282.336-.472.562-.57.676"
        stroke="#2e5f62"
        strokeWidth="1.572"
      />
      <path
        d="m3.66 19.946-.216.258c-.023.137-.006.228.049.274.055.046.148.047.28.002l.216-.257-.33-.277z"
        fill="#2e5f62"
        stroke="#2e5f62"
        strokeWidth=".943"
      />
      <path
        d="m16.384 3.954.79-.942c.469-.174.8-.18.993-.018.192.162.246.49.16.986l-.79.941-1.153-.967z"
        stroke="#2e5f62"
        strokeWidth="1.376"
      />
    </g>
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - listUnfinishedNote icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <svg
    data-cy="status-cell"
    height={24}
    viewBox="0 0 24 24"
    width={24}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M11.5278 3C10.6891 3 10.0134 3.68761 10.028 4.52613L10.1849 13.5261C10.1991 14.3442 10.8664 15 11.6846 15H12.3209C13.1392 15 13.8066 14.344 13.8206 13.5258L13.9752 4.52576C13.9896 3.68738 13.314 3 12.4754 3H11.5278ZM11.1173 17C10.501 17 10.0014 17.5373 10.0014 18.2V19.8C10.0014 20.4627 10.501 21 11.1173 21H12.8856C13.5019 21 14.0014 20.4627 14.0014 19.8V18.2C14.0014 17.5373 13.5019 17 12.8856 17H11.1173Z"
      fill="#ffa015"
      fillRule="nonzero"
      opacity="1"
    />
  </svg>
</div>
`;

exports[`Patient Status should match snapshot - selectedClose icon 1`] = `
<div
  className="statusCellstyled__StyledContainer-sc-3olh8y-1 eBriFQ"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <button
    className="iconstyled__StyledButton-ifk1iv-0 kgKJBV"
    onClick={[Function]}
  >
    <svg
      data-cy="status-cell"
      height={24}
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M7.70857 6.29289C7.31805 5.90237 6.68488 5.90237 6.29436 6.29289C5.90383 6.68342 5.90383 7.31658 6.29436 7.70711L10.5865 11.9993L6.2944 16.2914C5.90388 16.6819 5.90388 17.3151 6.2944 17.7056C6.68493 18.0961 7.31809 18.0961 7.70862 17.7056L12.0007 13.4135L16.2928 17.7056C16.6834 18.0961 17.3165 18.0961 17.7071 17.7056C18.0976 17.3151 18.0976 16.6819 17.7071 16.2914L13.4149 11.9993L17.7071 7.70711C18.0976 7.31658 18.0976 6.68342 17.7071 6.29289C17.3166 5.90237 16.6834 5.90237 16.2929 6.29289L12.0007 10.5851L7.70857 6.29289Z"
        fill="#999999"
        xmlns="http://www.w3.org/2000/svg"
      />
    </svg>
  </button>
</div>
`;
